.github/workflows/ci-docs-tests.yml

```diff
@@ -0,0 +1,154 @@
+# reference:
+#   - https://github.com/actions/cache
+#   - https://github.com/actions/checkout
+#   - https://github.com/marketplace/actions/setup-miniconda
+
+name: ci-docs-tests
+
+on:
+  push:
+    branches:
+      - "*" 
+
+  pull_request:
+    branches:
+      - "*" 
+
+jobs:
+  tests:
+    name: "doctests ${{ matrix.os }} ${{ matrix.python-version }}"
+
+    runs-on: ${{ matrix.os }}
+
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        python-version: ["3.8"]
+
+    env:
+      IRIS_TEST_DATA_VERSION: "2.9"
+      ENV_NAME: "ci-docs-tests"
+
+    steps:
+      - name: "checkout"
+        uses: actions/checkout@v3
+
+      - name: "environment configure"
+        env:
+          # Maximum cache period (in weeks) before forcing a cache refresh.
+          CACHE_WEEKS: 2
+        run: |
+          echo "CACHE_PERIOD=$(date +%Y).$(expr $(date +%U) / ${CACHE_WEEKS})" >> ${GITHUB_ENV}
+          echo "LOCK_FILE=requirements/ci/nox.lock/py$(echo ${{ matrix.python-version }} | tr -d '.')-linux-64.lock" >> ${GITHUB_ENV}
+
+      - name: "data cache"
+        uses: actions/cache@v3
+        id: data-cache
+        env:
+          # Increment the build number to force a cache refresh.
+          CACHE_BUILD: 0
+        with:
+          path: ~/iris-test-data
+          key: ${{ runner.os }}-iris-test-data-${{ env.ENV_NAME }}-v${{ env.IRIS_TEST_DATA_VERSION }}-b${{ env.CACHE_BUILD }}
```

Comment:
I didn't see the point, as for the data the key change is the `IRIS_TEST_DATA_VERSION`.

I don't see the point in purging a perfectly good cache, that hasn't change, because the period has lapsed.